scriptmodules/supplementary/skyscraper.sh

```diff
@@ -133,27 +144,30 @@ function configure_skyscraper() {
     # Handle the cases where the user has an existing Skyscraper installation.
     if [[ -d "$home/.skyscraper" && "$local_config" != "$configdir/all/skyscraper" ]]; then
         # We have an existing Skyscraper installation, but not handled by this scriptmodule.
-        # Since the $HOME/.skyscraper folder will be moved, make sure the Local DB and Import folders are moved separately
+        # Since the $HOME/.skyscraper folder will be moved, make sure the 'cache' and 'import' folders are moved separately
         local f_size
-        f_size=$(du --total -sm "$home/.skyscraper/dbs" "$home/.skyscraper/import" 2>/dev/null | grep total | cut -f 1 )
-        printMsgs "console" "INFO: Moving the Local DB and Import folders to new configuration folder (total: $f_size Mb)"
+        local cache_folder="dbs"
+        [[ -d "$home/.skyscraper/cache" ]] && cache_folder="cache"
+
+        f_size=$(du --total -sm "$home/.skyscraper/$cache_folder" "$home/.skyscraper/import" 2>/dev/null | grep total | cut -f 1 )
```

Comment:
total is always on the last line right ? So no need to grep every row. eg `| tail -n1 | cut -f 1` ?
`